jdaviz/core/helpers.py

```diff
@@ -402,6 +405,72 @@ def show_in_new_tab(self, title=None):  # pragma: no cover
                       DeprecationWarning)
         return self.show(loc="sidecar:tab-after", title=title)
 
+    def get_data(self, data_label, cls='default', subsets_to_apply=None, statistic=None):
```

Comment:
we could perhaps support `data_label=None` by default for the case of a single entry in `data_collection`, but then would need to add an extra check to raise an error if `data_label` is not passed and more than one entry exists.